src/main/kotlin/com/hypto/iam/server/validators/RequestModelValidator.kt

```diff
@@ -165,6 +166,12 @@ fun ResetPasswordRequest.validate(): ResetPasswordRequest {
 
 fun VerifyEmailRequest.validate(): VerifyEmailRequest {
     verifyEmailRequestValidation.validateAndThrowOnFailure(this)
+    Validation {
+        (this::oneOf)(
+            this@validate,
+            listOf(VerifyEmailRequest::email, VerifyEmailRequest::userHrn)
+        )
```

Comment:
Can this be moved inside `verifyEmailRequestValidation`?

Reply:
As discussed, this is not required. So resolving.